scripts/src/main/java/com/mathewsachin/fategrandautomata/scripts/locations/Locations.kt

```diff
@@ -21,7 +21,8 @@ class Locations @Inject constructor(
     val continueRegion = Region(120, 1000, 800, 200).xFromCenter()
     val continueBoostClick = Location(-20, 1120).xFromCenter()
 
-    val inventoryFullRegion = Region(-280, 860, 560, 190).xFromCenter()
+    val inventoryFullEnhancementButtonRegion = Region(-280, 860, 560, 190).xFromCenter()
+    val inventoryFullSellButtonRegion = Region(-880, 860, 560, 190).xFromCenter()
```

Comment:
If I understand correctly, you had to add this Region because the button is located somewhere else for FP summons?

Because all values are the same except for X, I combined both regions into one big one by extending the old one 600 to the left.